api/time_bucket.md

```diff
@@ -1,58 +1,59 @@
-## time_bucket() 
+# time_bucket()
+The `time_bucket` function is based on the standard PostgreSQL `date_trunc`
```

Comment:
This is not true and very misleading. We use our own implementation, completely independent from date_trunc.

Reply:
How would you reword that?

Reply:
The `time_bucket` function is somewhat similar to the standard PostgreSQL `date_trunc` function. Unlike `date_trunc`, ...